_posts/2022-05-27-gsoc-students.md

```diff
@@ -25,18 +25,22 @@ Add a link to your blog here!
 ## Bjarne Feddersen: Adding Energy Readers to MDAnalysis
 
 <img
-src="Bjarne's Picture Here"
+src="https://media-exp1.licdn.com/dms/image/C4D03AQHiLg970LOylQ/profile-displayphoto-shrink_800_800/0/1590149425816?e=1658966400&v=beta&t=455KVO-rbXV4YqtG7EeO4dQju3-Ar3GDMFtlpFP355I"
```

Comment:
I think we'll likely want this in the repo so we're not potentially linking to something else in the future.

Reply:
Good point - changed